schema/tables/cpu_info.yml

```
@@ -0,0 +1,13 @@
+name: cpu_info
+description: |- # (required) string - The description for this table. Note: this field supports Markdown
+	The `cpu_info` table expresses the data made available from a computer or mobile device by its Central Processing Unit (CPU).
+examples: |- # (optional) string - An example query for this table. Note: This field supports Markdown
+	Basic query:
```

@nonpunctual The YAML parser throws an error if a value starts with a tab character. If you change these lines to start with two spaces, it will pass the website test.

```suggestion
  The `cpu_info` table expresses the data made available from a computer or mobile device by its Central Processing Unit (CPU).
examples: |- # (optional) string - An example query for this table. Note: This field supports Markdown
  Basic query:
```